test/Api.IntegrationTest/SecretsManager/Controllers/ProjectsControllerTest.cs

```diff
@@ -134,34 +203,59 @@ public async Task UpdateProject_NotFound()
     }
 
     [Fact]
-    public async Task UpdateProject_MissingPermission()
+    public async Task Update_MissingAccessPolicy_Throws_NotFound()
     {
-        // Create a new account as a user
-        await LoginAsNewOrgUser();
+        var (org, _) = await _organizationHelper.Initialize(true, true);
+        var (email, _) = await _organizationHelper.CreateNewUser(OrganizationUserType.User, true);
+        await LoginAsync(email);
 
         var project = await _projectRepository.CreateAsync(new Project
         {
-            OrganizationId = _organization.Id,
+            OrganizationId = org.Id,
             Name = _mockEncryptedString
         });
 
-
-        var request = new ProjectUpdateRequestModel()
+        var request = new ProjectUpdateRequestModel
         {
             Name = "2.3Uk+WNBIoU5xzmVFNcoWzz==|1MsPIYuRfdOHfu/0uY6H2Q==|/98xy4wb6pHP1VTZ9JcNCYgQjEUMFPlqJgCwRk1YXKg=",
         };
 
         var response = await _client.PutAsJsonAsync($"/projects/{project.Id}", request);
 
-        Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+        Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+    }
+
+    [Theory]
+    [InlineData(false, false)]
+    [InlineData(true, false)]
+    [InlineData(false, true)]
+    public async Task Get_SmNotEnabled_NotFound(bool useSecrets, bool accessSecrets)
+    {
+        var (org, _) = await _organizationHelper.Initialize(useSecrets, accessSecrets);
+        await LoginAsync(_email);
+
+        var project = await _projectRepository.CreateAsync(new Project
+        {
+            OrganizationId = org.Id,
+            Name = _mockEncryptedString
+        });
+
+        var mockEncryptedString2 = "2.3Uk+WNBIoU5xzmVFNcoWzz==|1MsPIYuRfdOHfu/0uY6H2Q==|/98xy4wb6pHP1VTZ9JcNCYgQjEUMFPlqJgCwRk1YXKg=";
+        var request = new ProjectCreateRequestModel { Name = mockEncryptedString2 };
+
+        var response = await _client.PutAsJsonAsync($"/projects/{project.Id}", request);
+        Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
     }
 
     [Fact]
-    public async Task GetProject()
+    public async Task Get_Success()
```

Comment:
Just curious, why is this one named "Get_" when it's creating a project 🤔

Reply:
While we do create a project using the repository, we're testing the get endpoint hence the naming.